autopush/db.py

```diff
@@ -841,8 +841,13 @@ def register_user(self, data):
             # AWS.
             raise AutopushException("data is missing router_type "
                                     "or connected_at")
+        # Mobile users set this value to "0" which means that they are
+        # never dropped from DynamoDB (DDB only drops values that have a
+        # "reasonable" expiration date within 2 years or so of present.)
+        # We double the expiration to match the previous behavior of
+        # idle registrations lasting about 2 rotations.
         if "expiry" not in data:
-            data["expiry"] = _expiry(MAX_EXPIRY)
+            data["expiry"] = _expiry(MAX_EXPIRY * 2)
```

Comment:
Probably should have asked this during #1162, but now that we don't enforce the expiry on the router table, do we even need the field at all? I don't see anything actually being done w/ it..

Reply:
Excellent point. It should be removed so that it's not confusing folks. 

Thank you for pointing out the blindingly obvious. 

/me sighs heavily and goes about doing just that.